source/features/better-diff-selection.tsx

```diff
@@ -0,0 +1,31 @@
+import './better-diff-selection.css';
+
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function disableDiffSelection(event: Event): void {
+	const target = event.target as HTMLElement;
+	target.closest('tbody')!.dataset.rghSelect = target.closest('td:last-child') ? 'right' : 'left';
+}
+
+function restoreDiffSelection(): void {
+	select('[data-rgh-select]')?.removeAttribute('data-rgh-select');
+}
+
+function init(): void {
+	delegate(document.body, '.diff-table tr:not(.js-expandable-line) td:nth-child(even)', 'mousedown', disableDiffSelection);
+	document.body.addEventListener('mouseup', restoreDiffSelection);
+}
+
+void features.add(__filebasename, {
+	include: [
+		pageDetect.hasCode
```

Comment:
I think this could be further reduced. `hasCode` includes comments and files pages. Perhaps we only need this `include`:
```suggestion
		() => select.exists('meta[name="diff-view"][content="split"]')

```